synapse/lib/storm.py

```diff
@@ -1343,15 +1343,33 @@ def getArgParser(self):
             raise s_exc.StormRuntimeError(mesg='Tee command must take at least one query as input.',
                                           name=self.name)
 
-        async for node, path in genr:  # type: s_node.Node, s_node.Path
+        async def wrapgenr():
```

Comment:
We shouldn't have to use a generator wrapper here - we can do this tracking with a variable outside of the original loop; then we check if the ig-got-nodes variable got nodes, and if it did not, we execute the raw queries.  That avoids any possible confusion of a None or a (None, None) object from being injected into the storm pipeline.

Reply:
updated